src/engine/DrawManager.java

```diff
@@ -2612,70 +2612,150 @@ else if (0 < (255 - ((int)(System.currentTimeMillis() - ghostTImer) / 10)))
-
+			long passedTime = System.currentTimeMillis() - ghostTImer;
+			boolean timePassedisEven = passedTime % 2 == 0;
+			if(timePassedisEven){
+				for (int i = 0; i < increments.length; i++){
+					if (System.currentTimeMillis() - ghostTImer > 100 * i) {
+						this.drawEntity(SpriteType.Ghost, ghost1PostionX+increments[i] + i * 4, ghost1PostionY + i *4, 2, 2, new Color(255,255,255,255 - i * 30));
+						this.drawEntity(SpriteType.Ghost, ghost2PostionX+increments[i]+ i *4, ghost2PostionY+ i *4, 2, 2, new Color(255,255,255,255 - i * 30));
+						if (increments[i] <= -15 || increments[i] >= 15) adds[i] *= -1;
+						increments[i] += adds[i];
+					}
 				}
+				ghost1PostionY--; ghost2PostionY--;
 			}
 			else {
-				if(System.currentTimeMillis() - ghostTImer < 1000){
-					this.drawEntity(SpriteType.Ghost, ghost1PostionX, ghost1PostionY, 2, 2, Color.white);
-					this.drawEntity(SpriteType.Ghost, ghost2PostionX, ghost2PostionY, 2, 2, Color.white);
-
-				}
-				else if (System.currentTimeMillis() - ghostTImer < 2000){
-					this.drawEntity(SpriteType.Ghost, ghost1PostionX, ghost1PostionY, 2, 2, Color.white);
-					this.drawEntity(SpriteType.Ghost, ghost2PostionX, ghost2PostionY, 2, 2, Color.white);
-
-				}
-				else{
-					this.drawEntity(SpriteType.Ghost, ghost1PostionX, ghost1PostionY, 2, 2, Color.white);
-					this.drawEntity(SpriteType.Ghost, ghost2PostionX, ghost2PostionY, 2, 2, Color.white);
-
+				for (int i = 0; i < increments.length; i++) {
+					if (System.currentTimeMillis() - ghostTImer > 100 * i) {
+						this.drawEntity(SpriteType.Ghost, ghost1PostionX+increments[i] + i * 4, ghost1PostionY + i *4, 2, 2, new Color(255,255,255,255 - i * 30));
+						this.drawEntity(SpriteType.Ghost, ghost2PostionX+increments[i]+ i * 4, ghost2PostionY+ i *4, 2, 2, new Color(255,255,255,255 - i * 30));
+					}
 				}
 			}
+//			this.drawEntity(SpriteType.Ghost, ghost1PostionX+tmp, ghost1PostionY, 2, 2, Color.white);
+//			this.drawEntity(SpriteType.Ghost, ghost2PostionX+tmp, ghost2PostionY, 2, 2, Color.white);
+//
+//			if (System.currentTimeMillis() - ghostTImer > 100){
+//				this.drawEntity(SpriteType.Ghost, ghost1PostionX+tmp-2, ghost1PostionY+2, 2, 2, new Color(255,255,255,200));
+//				this.drawEntity(SpriteType.Ghost, ghost2PostionX+tmp-2, ghost2PostionY+2, 2, 2, new Color(255,255,255,200));
+//			}
+//			if (System.currentTimeMillis() - ghostTImer > 200){
+//				this.drawEntity(SpriteType.Ghost, ghost1PostionX+tmp-4, ghost1PostionY+4, 2, 2, new Color(255,255,255,170));
+//				this.drawEntity(SpriteType.Ghost, ghost2PostionX+tmp-4, ghost2PostionY+4, 2, 2, new Color(255,255,255,170));
+//			}
+//			if (System.currentTimeMillis() - ghostTImer > 300){
+//				this.drawEntity(SpriteType.Ghost, ghost1PostionX+tmp-6, ghost1PostionY+6, 2, 2, new Color(255,255,255,140));
+//				this.drawEntity(SpriteType.Ghost, ghost2PostionX+tmp-6, ghost2PostionY+6, 2, 2, new Color(255,255,255,140));
```

Comment:
[P2]

If it don't need, it would be better delete this comment.
It makes it harder to read.

Reply:
If it's not your code, feel free to leave it for now.

Reply:
Thank you for your comment. I didn't write that, so we should leave it for now.